plaso/parsers/santa.py

```diff
@@ -20,11 +20,14 @@ class SantaExecutionEventData(events.EventData):
     action (str): action recorded by Santa.
     decision (str): if the process was allowed or blocked.
     reason (str): reason behind santa decision to execute or block a process.
+    explain (str): further explanation behind santa decision to execute
+        or block a process.
     process_hash (str): SHA256 hash for the executed process.
     certificate_hash (str): SHA256 hash for the certificate associated with the
         executed process.
     certificate_common_name (str): certificate common name.
     pid (str): process id for the process.
+    pid_version (str): process id version.
```

Comment:
id => identifier

I know this is not directly related to your changes, but please update the other ones in this docstring as well.

Reply:
Updated docstrings